packages/search-ui-app-search-connector/src/buildResponseAdapterOptionsFromConfig.js

```diff
@@ -0,0 +1,22 @@
+export default function buildResponseAdapterOptionsFromState(config) {
```

Comment:
Naive question here - I'm struggling a bit to understand how this can be expanded to apply to non-geo-specific facets. The name of the function seems to imply that it _generally_ builds options/configs, but the code itself seems pretty limited to geo `unit`/`center` fields.

Do you mind giving me an example of how this might be expanded to cover other facet configs? Or if we don't plan on expanding this down the road, should we maybe consider renaming this something like `buildGeoOptions` or something more specific?

Reply:
I can rename it to `buildGeoOptions`.

Reply:
Right now it's just a generic function. It takes config, and spits out additional options that need to be passed to the adapter. Down the road, it's possible we may need to pass additional config to the adapter, but as you point out, we don't need to right now.

The code would probably read better if it was just focused on geo options, but I think it would take another step or two of refactoring if we wanted to rename the function to be geo specific, and I frankly don't want to spend the cycles on that right now. Feel free to disagree if you feel strongly.

Reply:
Nope, I'm good with it! After looking at it a bit more, I also think I see how we could add other branches/if catches! :)